### ISSUE TITLE
How to get the 4h candles as 9:30-11:00-15:00 instead of 13:30 and 16:00?

### ISSUE THREAD

**Issue by alexh0101** [tedchou12/webull#385]
When I download 4h candles, I get only 2 candles per trading session (I don't want extended hours).

Is there a way I can get those 3 candles: 9:30-11:00-15:00?

**Comment by brycen-l** [comment]
I don't think so. A simple, or maybe complicated way would be to get the 2h candles and then convert these to what you want.

**Comment by LukeWang01** [comment]
Can we get hourly bars? 
we I call this:
`bar_data = wb.get_bars('QQQ', interval='h2', count=1000, extendTrading=0, timeStamp=None)`


I get this output, only one bar...
![image](https://user-images.githubusercontent.com/25569658/231334100-6abc57a2-5e5b-4071-bd7d-578833f0e83b.png)
